tree/tree/inc/TBranchCacheInfo.h

```diff
@@ -138,20 +138,26 @@ class TBranchCacheInfo {
    }
 
    /// Print the info we have for the baskets.
-   void Print(const char *owner, Long64_t *entries) const
+   void Print(const char *owner, Long64_t *entries, Int_t nBaskets, Long64_t nEntries) const
    {
       if (!owner || !entries)
          return;
       auto len = fInfo.GetNbits() / kSize + 1;
-      if (fBasketPedestal >= 0)
+      if ( (fBasketPedestal + len) > (UInt_t)nBaskets)
```

Comment:
Apparently `fBasketPedestal` can be negative (I guess it can be from {-1, 0, ..., 3}?) but is here compared with the unsigned version of `nBaskets`. Is this wanted?

Reply:
Correct.  I update the code to return immediately if pedestal is negative.